### ISSUE TITLE
error 230 on list

### ISSUE THREAD

**Issue by bzuidgeest** [ArxOne/FTP#43]
Hello,

I was trying out the library and noted the following:

When connecting to speedtest.tele2.net I could do a listing no problems. However this server does not support mlsd. So I tried ftp.cs.brown.edu. I verified connection and mlsd support using filezilla. But when I connect using this library I get a ftpprotocolexception: "Expected other reply than 230 ('Anonymous user logged in')" on normal and mlsd listing. 

This problem might just be related to brown.edu ftp software and I am just using it as a test server. But I thought to write this down here, as it might affect others to and could be as simple fix.

My test code
```
using (var ftpClient = new FtpClient(new Uri("ftp://speedtest.tele2.net"), null, new FtpClientParameters()
//using (var ftpClient = new FtpClient(new Uri("ftp://ftp.cs.brown.edu"), null, new FtpClientParameters()
            {
                Passive = false,
            }))
            {
                var temp = ftpClient.ListEntries(""); //.MlsdEntries(""); //.Mlsd(new FtpPath("/"));
            }
```

**Comment by picrap** [comment]
Hi,

I wrote a fix for this. It is available in new version 1.14.1.

Thanks.

**Comment by bzuidgeest** [comment]
Just finished testing with the new release. Works perfectly, thanks for fixing it so quickly.
